frontend/app/components/Thermometer.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import styles from "./styles/Thermometer.module.scss";
+
+type Props = {
+  tasks: string;
+};
+
+const Thermometer: React.FC<Props> = ({ tasks }) => {
+  const taskStatusInPercent = { complete: 10, asPlanned: 30, unknown: 25, delayed: 30, failed: 5 };
+  const taskStatusInOrder = [
+    taskStatusInPercent.failed,
+    taskStatusInPercent.delayed,
+    taskStatusInPercent.unknown,
+    taskStatusInPercent.asPlanned,
+    taskStatusInPercent.complete,
+  ];
+  const taskStatusCumulative = taskStatusInOrder.map(
+    (
+      (sum) => (value) =>
+        (sum += value)
+    )(0),
+  );
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.thermometer}>
+        <div
+          style={{ bottom: taskStatusCumulative[4] + "%", height: taskStatusInOrder[4] + "%" }}
```

Comment:
bottom has no effect here as the position is not absolute or relative. Aber geht auch ohne da die Höhe ja prozentual ist